server/watson-discovery-service.js

```diff
@@ -5,8 +5,8 @@ const vcapServices = require('vcap_services');
 const discoveryCredentials = vcapServices.getCredentials('discovery');
 
 const discovery = new DiscoveryV1({
-  username: discoveryCredentials.username,
-  password: discoveryCredentials.password,
+  username: process.env.DISCOVERY_SERVICE_USERNAME || discoveryCredentials.username,
```

Comment:
NOTE that this only works if you rename the vars in .env to DISCOVERY_USERNAME and DISCOVERY_PASSWORD. I have these changes in patch 10.